napari/resources/__init__.py

```diff
@@ -71,7 +71,13 @@ def import_resources(version: str = '', overwrite: bool = False) -> None:
             - the resources file does not already exist.
             - ``overwrite`` argument is True
             - the ``NAPARI_REBUILD_RESOURCES`` environmental variable is set
-    
+
+    Returns
+    -------
+    out_path : str
+        Path to the python resource file.  Import this file to make the SVGs
```

Comment:
can you change this docstring to indicate that the file _is_ imported by this function.  This makes it sound like you need to import the file (after running this func) to make the SVGs available.

Reply:
Please check new version.